e2e-tests/ts-client/setup/setupFishjam.ts

```diff
@@ -0,0 +1,23 @@
+import { DockerComposeEnvironment, Wait } from 'testcontainers';
+import { setupState } from './globalSetupState';
+import { type NetworkInterfaceInfo, networkInterfaces } from 'os';
+
+export default async function setupFishjam() {
+  const EXTERNAL_IP = Object.values(networkInterfaces())
+    .flat()
+    .filter((x): x is NetworkInterfaceInfo => x !== undefined)
```

Comment:
would `.filter(Boolean)` do the job? ;p

Reply:
It might work as above from TS 5.5 (we are using 5.4).